grid2op/Episode/EpisodeData.py

```diff
@@ -159,6 +159,11 @@ def __init__(self,
                                              helper_action_env,
                                              "env_actions",
                                              check_legit=False)
+
+        self.attacks = CollectionWrapper(attack,
+                                         attack_space,
+                                         "attacks")
+
```

Comment:
Try to document this feature in the doc please :-)

Reply:
Done in abdd653d. I documented the attributes observations, actions, env_actions and attacks that all follow the same logic. The whole class EpisodeData needs documentation though. I can probably do that in the following weeks.